include/openssl/pem.h

```diff
@@ -489,6 +489,13 @@ OPENSSL_EXPORT EC_GROUP *PEM_read_bio_ECPKParameters(BIO *bio,
 OPENSSL_EXPORT int PEM_write_bio_ECPKParameters(BIO *out,
                                                 const EC_GROUP *group);
 
+// PEM_write_bio_PrivateKey_traditional calls |PEM_ASN1_write_bio| to write
+// out |x|'s private key in the "traditional" ASN1 format. Use
+// |PEM_write_bio_PrivateKey| instead.
+OPENSSL_EXPORT int PEM_write_bio_PrivateKey_traditional(
```

Comment:
should this be `OPENSSL_DEPRECATED`?

Reply:
I was weighing it, but the API naming seemed like OpenSSL was planning to maintain this long term. It's not deprecated in 3.x, so I decided to not mark it.